doc/source/methods/PermutationImportance.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Rename <code>metric_fns</code> to <code>metric_fn</code> since it's a single one?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/798/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>